packages/form-js-editor/test/spec/properties-panel/PropertiesPanel.spec.js

```diff
@@ -575,6 +580,108 @@ describe('properties panel', function() {
 
       });
 
+
+      describe('dynamic values', function() {
+
+        it('source should be configurable', function() {

```

Comment:
Maybe rephrase, as it (also) tests a behavior.
```suggestion
        it('should be configurable - sets valuesKey', function() {

```

Reply:
The same would go for the other test case below.